### ISSUE TITLE
Tooltip Jitter On Initial Render

### ISSUE THREAD

**Issue by tony-cocco** [plouc/nivo#2161]
**Describe/explain the bug**
Initial tooltip render appears to the bottom right of the cursor before snapping into place.

**To Reproduce**
https://codesandbox.io/s/nivo-tooltip-flicker-consistency-yoln21?file=/src/index.js

Issue can be seen in the codesandbox. The top chart tooltip flickers when entering the chart while the bottom one doesn't. This is also true when no custom tooltip property is set. Both divs are rendering the same pie chart just using a different method. I've also confirmed the issue still happens when only 1 chart is present (the rendering method is the only thing that changes the behavior). This flicker is also present in my nextjs application.

1. Go to sandbox link
2. View tooltip on top chart
3. View tooltip on bottom chart

**Expected behavior**
Both tooltips should function the same.

**Screenshots**

https://user-images.githubusercontent.com/4268000/195429258-9ae4fad4-1f76-4b1b-9fd5-a186df0efc9b.mov

**Desktop (please complete the following information):**
 - OS: Windows 10 and OS 12.6
 - Browser:  Chrome
 - Version: Version 106.0.5249.119 (Official Build) (64-bit), Version 105.0.5195.125 (Official Build) (x86_64)

**Additional context**
Hopefully this is maybe a silly config issue somewhere, but I could not identify any difference other than render method. The codesandbox console does display `Warning: ReactDOM.render is no longer supported in React 18...`, which appears to be from the bottom chart that is working correctly.

**Comment by posabsolute** [comment]
We got the same issue in our app, it flickers for like 100ms.

**Comment by evaK-de** [comment]
Has anybody found a fix for this? It is pretty annoying and React 18 has been out for quite a while now.

**Comment by posabsolute** [comment]
I had to hide it for the first few MS.

**Comment by avgupta456** [comment]
I'm also running into this issue... Could you share a bit on how you managed to hide it for the first few ms?

**Comment by CyrusNG** [comment]
Another way to hotfix it, make tooltip root <div> position as absolute, and set width at the same time:

```
tooltip={(bar) => (
  <div className="position-absolute" style={{"min-width":"120px", "transform":"translate(-50%, -100%)"}} key={bar.id}>
    <strong>Hello World</span>
  </div>
)}
```

**Comment by robin-leach** [comment]
bump

**Comment by EmilianoMorghen** [comment]
bump

**Comment by arackaf** [comment]
> I guess this repo is not maintained anymore so I'm not sure that bug will be fixed any time soon

Can anyone confirm that? From most of the issues I've seen, it's one dude managing this project. Has he just moved on? Understandable if so, but good to know.

Most recent commits appear to be from 4 months ago.

**Comment by posabsolute** [comment]
Looks like it is still somewhat maintained, @plouc seems to be the main dev, but I am guessing someone would need to create a patch to get it merged in.

**Comment by DimaAmega** [comment]
He guys! Fixed it here in #2480
